build/notice-file/Readme.md

```diff
@@ -0,0 +1,34 @@
+# Notice.txt File Configuration
+
+We are automatically generating Notice.txt by using first-level dependencies of the project. The related pipeline uses `config.yaml` stored in this folder.
+
+
+## Configuration
+
+Sample:
+
+```
+title: "Mattermost Mobile"
+copyright: "© 2016-present Mattermost, Inc.  All Rights Reserved.  See LICENSE.txt for license information."
+description: "This document includes a list of open source components used in Mattermost Mobile, including those that have been modified."
+reviewers: 
+  - mattermost/release-managers
+  - enahum
+search:
+  - "package.json"
+dependencies:
+  - "wix"
+devDependencies: 
+  - "webpack"
```

Comment:
???

Reply:
Those are mentioned as samples in the Readme.md. The goal is to have a automated Notice.txt. Those are not needed in mobile project.

We need to include wix's licence into the Notice.txt file even if it is not a first level dependency, this is requested by desktop project.

And for webpack, webapp requested that, webpack dev depedency licence need to be included in the Notice.txt.

To make that configurable, we created two section. I will improve Readme on those fields.